src/main/java/it/gov/pagopa/reward/service/lookup/HpanInitiativeMediatorServiceImpl.java

```diff
@@ -136,23 +149,49 @@ private Flux<HpanUpdateEvaluateDTO> initializingHpanInitiativeDTO(HpanInitiative
     }
 
     private Mono<String> findAndModify(HpanUpdateEvaluateDTO hpanUpdateEvaluateDTO) {
+        BaseOnboardingInfo baseOnboardingInfo = new BaseOnboardingInfo(hpanUpdateEvaluateDTO.getInitiativeId(), null);
```

Comment:
familyId is null if onboarding new initiative (maybe neither the counter is created?)